Base/usr/share/man/man1/su.md

```diff
@@ -0,0 +1,41 @@
+## Name
+
+su - run a command with substitute user
+
+## Synopsis
+
+```
+$ su [user]
+```
+
+## Description
+
+**su** allows commands to be run with a substitute user.
+
+When called with no user-specified, **su** defaults to running an interactive shell as *root*. 
+
+## Options
+
+- **--help**
+
+  Display help message and exit.
+
+- **--version**
+
+  Display version information and exit.
+
+## Arguments
+
+- user: User to switch to (defaults to user with UID 0)
+
+## Examples
+
+```
+# Switch to root user
+$ su
+
+#Switch to another user

```

Comment:
Space here after the `#`, let's be consistent with the other comment three lines above :P

Reply:
Sure, I will modify it right now.

Reply:
I have modified the file, but the lint_commits kept failing, can I open another PR to contribute the su.md file.